src/Kafka/Consumer/Types.hs

```diff
@@ -30,14 +30,17 @@ where
 
 import Data.Bifoldable      (Bifoldable (..))
 import Data.Bifunctor       (Bifunctor (..))
-import Data.Bitraversable   (Bitraversable (..), bimapM, bisequenceA)
+import Data.Bitraversable   (Bitraversable (..), bimapM, bisequence)
 import Data.Int             (Int64)
 import Data.Text            (Text)
 import Data.Typeable        (Typeable)
 import GHC.Generics         (Generic)
 import Kafka.Internal.Setup (HasKafka (..), HasKafkaConf (..), Kafka (..), KafkaConf (..))
 import Kafka.Types          (Millis (..), PartitionId (..), TopicName (..))
 
+-- | The main type for Kafka consumption, used e.g. to poll and commit messages.
+-- 
+-- Its constructor is intentionally not exposed, instead, one should used 'newConsumer' to acquire such a value.
```

Comment:
I felt this was useful in understanding the module mechanics to explain the constructor is hidden + link to how to create such a value